app/views/kaminari/_gap.html.slim

```diff
@@ -0,0 +1,2 @@
+li.disabled
+  = link_to raw(t 'views.pagination.truncate'), '#'
```

Comment:
To be honest, I don't really know, these are the views generated by Kaminari with the Bootstrap style and the `slim` format. Didn't really mess with them much :D